lru_test.go

```diff
@@ -8,63 +8,94 @@ import (
 	"testing"
 )
 
-func BenchmarkLRU_Rand(b *testing.B) {
-	l, err := New[int64, int64](8192)
-	if err != nil {
-		b.Fatalf("err: %v", err)
-	}
+func Benchmark_Rand(b *testing.B) {
+	var fn = func(b *testing.B, l *Cache[int64, int64]) {
+		trace := make([]int64, b.N*2)
+		for i := 0; i < b.N*2; i++ {
+			trace[i] = getRand(b) % 32768
```

Comment:
I'd get rid of getRand() and its accompanying file (it only contains this function).  Instead I would use that function once to generate several random lists of values.  

I would then use them in tests instead of random generation every time.  This just gives more consistent results and if you do it over say 10 or 100 random set of data, this is likely to give you more of what you are looking for.  It also helps track down weaknesses from a particular set of values for an algorithm, which can't be done with this method.  And you are doing go get an apples to apples on benchmark runs.

Reply:
I will skip this comment in the current  iteration. Will address them in a subsequent PR.